azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/controlbar/ControlBarView.kt

```diff
@@ -117,6 +117,12 @@ internal class ControlBarView : LinearLayout {
                     R.drawable.azure_communication_ui_ic_fluent_speaker_2_24_regular_composite_button_filled
                 )
             }
+            AudioDeviceSelectionStatus.BLUETOOTH_SCO_SELECTED -> {
+                callAudioDeviceButton.setImageResource(
+                    // Needs an icon
+                    R.drawable.azure_communication_ui_ic_fluent_speaker_2_24_regular_composite_button_filled
```

Comment:
Please reach out to AlexP to obtain vector icons for both selected and unselected state